src/lib.c

```diff
@@ -455,10 +480,42 @@ void calc_per_store(cart_item_s cart_item[], int n_shopping_list, int n_stores,
 
     qsort(cart, n_stores, sizeof(cart_sum_s), compare_cart);
 
-    //print function
+
+    //print function, prints the store name, prints the distance to the store, prisen for distance, prints the item expenesses og summert af item expenses og travel exprenses
     for (int i = 0; i < n_stores; i++) {
-        printf("|Name > %8s : Distance > %4d : Total sum > %4.2lf|\n", cart[i].store.name,
-               calc_base_to_store(cart[i].store), cart[i].sum);
+        printf("|Name > %8s : Distance > %4d Travel expenses > %5.2lf: Item expenses > %4.2lf Total sum %4.2lf|\n", cart[i].store.name,
+               calc_base_to_store(cart[i].store),calc_gas_price(km_price, calc_base_to_store(cart[i].store)), cart[i].sum, calc_gas_price(km_price, calc_base_to_store(cart[i].store)) + cart[i].sum);
+    }
+    // could add an if statement, so it doesnt print travel expenses.
+}
+
+void user_input(char user_location_f[], int* user_location, double* user_lat,double* user_lon, double* km_price, int* radius, int* transport){
+
+    FILE* f = open_file(user_location_f);
+    int current_location; //select from preivously loaded locations
+    char by_car = 0;
+    printf("Please select your location. \n Locations available: '1' school, '2' home >\n");
```

Comment:
Ved ikke om vi skal holde os til 1 lokation, da "dokumentationen" af hvordan der skal stå 1 og 2 foran hver af lokationerne og det er hardcoded school og home er lidt wack. Kan ikke helt se argumentet for det.